### DISCUSSION TITLE
Getting error while trying to import ross in my code

### DISCUSSION THREAD

**Discussion by SaeedHekmat** [petrobras/ross#996] [Q&A]
Hello guys;
I recently ran into this package and I'm interested to use it. I used pip to install it but when I try to import it in my code I get the error: No module named 'ross' 
It is worth noting that I code in Spyder. I knew that I should have used "Anaconda Prompt" to install it but I ran into errors when trying to install it using this : "conda install ross-rotordynamics"
Can anyone help me with that?

**Comment by raphaeltimbo** [comment]
Hi @SaeedHekmat! 
It looks like this could be an issue with your python environment.
Do you have different python installations in your computer?
Also, even when using the anaconda prompt to install ROSS you should still use `pip install ross-rotordynamics`. The recommendation of using the anaconda prompt is just too be sure that you are installing the ROSS package in your anaconda environment.
